### PR TITLE
Remove title check from conventional commits, resolves #14

### LINKED ISSUE #14: 🛠 Tooling: Remove compliance action PR title checks?
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/philly-js-club/philly-js-club-website-remix/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

Nobody other than me has been using semantic PR titles (`chore: ...`, `feat: ...`, etc.). And because this repo doesn't have semantic-versioned releases (`1.2.3`, etc.), there's really not much _technical_ benefit to using them... Maybe we should remove that part of PR checks?

### Additional Info

The action is from here: https://github.com/mtfoley/pr-compliance-action/

### PR DESCRIPTION
**Pull request by seriouslysean**
## PR Checklist

- [x] Addresses an existing open issue: fixes #014
- [x] That issue was marked as [`status: accepting prs`](https://github.com/philly-js-club/philly-js-club-website-remix/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/philly-js-club/philly-js-club-website-remix/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Removed the title check, so that titles no longer need to conform to conventional commits guidelines. Resolves #14.

### CODE REVIEW

File: .github/workflows/compliance.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ jobs:
             renovate
             renovate[bot]
           ignore-team-members: false
+          title-check-enable: false
```
Comment (seriouslysean) [inline .github/workflows/compliance.yml:14]: https://github.com/mtfoley/pr-compliance-action#inputs
image

**Comment by seriouslysean** [comment]
Removed `feat: ` from the title to see if it would still pass compliance to be sure, https://github.com/philly-js-club/philly-js-club-website/actions/runs/4851503565/jobs/8645409640?pr=21. ✅

**Review by JoshuaKGoldberg — APPROVED** [review]
💯 thanks!

**Comment by JoshuaKGoldberg** [comment]
@allcontributors add @seriouslysean for code

**Comment by JoshuaKGoldberg** [comment]
@allcontributors add @seriouslysean for code

**Pull request merged**